src/api/Configuration.ts

```diff
@@ -30,6 +30,79 @@ export namespace GlobalConfiguration {
   }  
 }
 
+export interface StoredConnection {
+  index: number,
+  data: ConnectionData
+};
+
+export namespace ConnectionManager {
+  export function getByName(name: string): StoredConnection | undefined {
+    const connections = getAll();
+    if (connections) {
+      const index = connections.findIndex(conn => conn.name === name);
+      if (index !== -1) {
+        return { index, data: connections[index] };
+      }
+    }
+  }
+
+  export function sort() {
+    const connections = getAll();
+    connections.sort((a, b) => a.name.localeCompare(b.name));
+    return GlobalConfiguration.set(`connections`, connections);
+  }
+
+  export function getAll(): ConnectionData[] {
+    return GlobalConfiguration.get<ConnectionData[]>(`connections`) || [];
+  }
+
+  function setAll(connections: ConnectionData[]) {
+    return GlobalConfiguration.set(`connections`, connections);
+  }
+
+  export async function storeNew(data: ConnectionData): Promise<StoredConnection> {
+    let connections = getAll();
```

Comment:
Never use `let` unless it's absolutely necessary 😉
```suggestion
    const connections = getAll();
```